### PR TITLE
Add additional intiatives

### PR DESCRIPTION
**Pull request by captainsafia**
Initial drafts of more initiatives.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ that the initiative is pending completion of the roadmap, grant funding, or othe
 | [nteract desktop application beta](https://github.com/nteract/nteract/blob/master/ROADMAP.md) | UNDER DEV  |
 | [Remote Kernels](./remote-kernels)                              | PENDING    |
 | [React Components](./remote-components)                         | PENDING    |
+| [Bundled Application](./bundled-app)                            | UNDER DEV  |
+| [Real-time Collaboration](./real-time-collab)                   | PENDING    |
+| [git Integration](./git-integration)                            | PENDING    |
```
Comment (rgbkrk) [inline README.md:23]: 👍 to these

File: bundled-app/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Roadmap for Bundled Application
+
+A large majority of interactive notebook users use notebooks against a
+traditional SciPy stack. Sometimes installing this talk can be difficult.
```
Comment (rgbkrk) [inline bundled-app/README.md]: s/talk/stack

File: bundled-app/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+Although tools like `conda` make the process easier it can still be a
+hassle for some users. Our bundled kernel intiative aims to make this
+easier.
```
Comment (rgbkrk) [inline bundled-app/README.md]: The main drive here is to make the barrier to entry really low. Install and run one thing, show people the way to get more complex later on.

File: bundled-app/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+With a bundled kernel, users can download a version of the nteract application
+with a bundled conda environment for their system architecture. This reduces
+the hassle for users that just want to get up and running with a
+simple SciPy stack.
```
Comment (rgbkrk) [inline bundled-app/README.md:15]: 👍

File: git-integration/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Roadmap for Git Integration
+
+Many notebook users utilize git and GitHub to manage versions of
+their notebooks and share them with others. Sometimes this process
+can be quite difficult as creating diffs of the notebook format
+comes with a lot of challenges. We are hoping to address this
+issue and create tightly-coupled git integration in the nteract
+desktop application.
```
Comment (rgbkrk) [inline git-integration/README.md]: I'd distill this into these bullet points:
- Ability to commit straight from the notebook interface
- Visual representation of diffs

We can't solve the issue of the notebooks themselves being difficult to handle, luckily the diffing tools are being worked on by others including @minrk.

File: real-time-collab/README.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Roadmap for Real-time Collaboration
+
+Reearch and data science teams use and share the notebook amongst
```
Comment (rgbkrk) [inline real-time-collab/README.md]: s/Reearch/Research/

File: real-time-collab/README.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+each other in order to complete their work. However, collaborating
+on the same notebook at the same time is a cubersome process. Our goal
```
Comment (rgbkrk) [inline real-time-collab/README.md]: s/cubersome/cumbersome

**Comment by rgbkrk** [comment]
Sweet. We can keep updating these too. Thanks for penning these.

File: git-integration/README.md (hunk 5 above)
Comment (minrk) [reply git-integration/README.md]: You could add some things for the opinionated stance of storing notebooks in lfs, or enabling UI for actions that some people have requested, e.g. strip-on-commit. Hopefully the nbdime diff rendering should be usable in an nteract context once it's more polished.

Comment (captainsafia) [reply git-integration/README.md]: Noted! I'll probably add more technical context to these initiatives sometime this week and will ping you to review the one on git integration.

**Pull request merged**